<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirCoolControllClass</class>
 <widget class="QMainWindow" name="AirCoolControllClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>685</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>685</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AirCoolControll</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect_to_device"/>
   <addaction name="actionDisconnect_from_UART"/>
   <addaction name="actionConnect_to_host"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPreferences"/>
  </widget>
  <action name="actionConnect_to_device">
   <property name="icon">
    <iconset resource="aircoolcontroll.qrc">
     <normaloff>:/Images/connect.png</normaloff>:/Images/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect_to_device</string>
   </property>
   <property name="toolTip">
    <string>Connect to device</string>
   </property>
  </action>
  <action name="actionDisconnect_from_UART">
   <property name="icon">
    <iconset resource="aircoolcontroll.qrc">
     <normaloff>:/Images/disconnect.png</normaloff>:/Images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect_from_UART</string>
   </property>
   <property name="toolTip">
    <string>disconnect line</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="aircoolcontroll.qrc">
     <normaloff>:/Images/preferences.png</normaloff>:/Images/preferences.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>settings menu</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionConnect_to_host">
   <property name="icon">
    <iconset resource="aircoolcontroll.qrc">
     <normaloff>:/Images/connect_to_host.png</normaloff>:/Images/connect_to_host.png</iconset>
   </property>
   <property name="text">
    <string>connect_to_host</string>
   </property>
   <property name="toolTip">
    <string>Connect ot remote host</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="aircoolcontroll.qrc"/>
 </resources>
 <connections/>
</ui>
